web_tree_dynamic_colored_field/README.rst

```diff
@@ -0,0 +1,133 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+============================
+Colorize field in tree views
+============================
+
+This module aims to add support for dynamically coloring fields in tree view
+according to data in the record.
+
+It provides attributes on fields with the similar syntax as the ``colors`` attribute
+in tree tags.
+
+Further, it provides a ``color_field`` attribute on tree tags's ``colors`` to use
+a field's value as color.
+
+Features
+========
+
+* Add attribute ``bg_color`` on field's ``options`` to color background of a cell in tree view
+* Add attribute ``fg_color`` on field's ``options`` to change text color of a cell in tree view
+* Add attribute ``color_field`` on the tree element's ``colors`` to use as color
+
+Usage
+=====
+
+* In the tree view declaration, put ``options='{"bg_color": "red: customer==True"}`` attribute in the ``field`` tag::
```

Comment:
Please also add an example to use char type of field (selection) in the condition.
Ex: options='{"bg_color": "red: state == 'rejected'' "}

Reply:
Any clue on this?
@mreficent